### DISCUSSION TITLE
I’d like to ask if it's possible to modify the shortcut for switching between two tabs. I noticed the default is control+tab.

### DISCUSSION THREAD

**Discussion by oushiei120** [zen-browser/desktop#3343] [General]
I’d like to ask if it's possible to modify the shortcut for switching between two tabs. I noticed the default is control+tab.

**Comment by StDensity** [comment]
In MS Edge, you can do it with Alt + Tab, it's really convenient because at the same time you can switch between recent tabs and other programs. It would be cool to have that.
